src/rust-crypto/rust-crypto.ts

```diff
@@ -766,8 +768,8 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, RustCryptoEv
      */
     public async getSessionBackupPrivateKey(): Promise<Uint8Array | null> {
         const backupKeys: RustSdkCryptoJs.BackupKeys = await this.olmMachine.getBackupKeys();
-        if (!backupKeys.decryptionKeyBase64) return null;
-        return Buffer.from(backupKeys.decryptionKeyBase64, "base64");
+        if (!backupKeys.decryptionKey) return null;
+        return Buffer.from(backupKeys.decryptionKey.toBase64(), "base64");
```

Comment:
strictly speaking this isn't required but https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/15 deprecates `decryptionKeyBase64` and I was in the area.